README.md

```diff
@@ -12,13 +12,19 @@ Examples such as the Hero above can be found in the `examples` folder. To run th
 
 ### Editor
 
-Add the files `lib/playcanvas-spine.3.6.min.js` and `lib/spine.js` to your project. Note that a 3.8 version of the library is available too.
+Add the plugin matching the Spine version used to export the animations, i.e `build/playcanvas-spine.X.X.min.js` and the Playcanvas script `build/spine.js` to your project. The following plugins are available:
+
+| Spine Editor | PlayCanvas Engine |
+| ------------ | ----------------- |
+| 3.6          | Up to 1.65        |
```

Comment:
Can 3.6 and 3.8 still be used with newer engine?

Reply:
Yes, they work with warnings on Mesh see #70.  The rewrite might address this.

Reply:
Engine support is a best guess from looking at commit history.